src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,343 @@
+    private static final String OVERWRITE_WORKSPACE_NAME = "CreateGenomicsDBWorkspace";
+
+    @Argument(fullName = WORKSPACE_ARG_NAME,
+              shortName = WORKSPACE_ARG_NAME,
+              doc = "Workspace for GenomicsDB. Has to be a POSIX file system path")
+    private String workspace;
+
+    @Argument(fullName = ARRAY_ARG_NAME,
+              shortName = ARRAY_ARG_NAME,
+              doc = "TileDB array name used by GenomicsDB. Defauts to " + ARRAY_ARG_NAME,
+              optional = true)
+    private String arrayName = DEFAULT_ARRAY;
+
+    @Argument(fullName = SEGMENT_SIZE_ARG_NAME,
+              shortName = SEGMENT_SIZE_ARG_NAME,
+              doc = "Buffer size in bytes allocated for GenomicsDB attributes during " +
+                    "import. Should be large enough to hold data from one site. " +
+                    " Defaults to " + DEFAULT_SEGMENT_SIZE,
+              optional = true)
+    private long segmentSize = DEFAULT_SEGMENT_SIZE;
+
+    @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME,
+              shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME,
+              doc = "GVCF files to be imported to GenomicsDB. Each file must contain" +
+                    "data for only a single sample")
+    private List<String> variantFilePaths;
+
+    @Argument(fullName = VCF_BUFFER_SIZE_ARG_NAME,
+              shortName = VCF_BUFFER_SIZE_ARG_NAME,
+              doc = "Buffer size in bytes to store variant contexts." +
+                    " Larger values are better as smaller values cause frequent disk writes." +
+                    " Defaults to " + DEFAULT_VCF_BUFFER_SIZE_PER_SAMPLE,
+              optional = true)
+    private long vcfBufferSizePerSample = DEFAULT_VCF_BUFFER_SIZE_PER_SAMPLE;
+
+    @Argument(fullName = CALLSET_MAP_FILE_ARG_NAME,
+              shortName = CALLSET_MAP_FILE_ARG_NAME,
+              doc = "Path to callset JSON file to be created. " +
+                    "The file contains row mappings to sample names."
+                    + " Default " + DEFAULT_CALLSETMAP_FILE_NAME +
+                    " file is written to GenomicsDB workspace",
+              optional = true)
+    private String callsetMapJSONFileName = DEFAULT_CALLSETMAP_FILE_NAME;
+
+    @Argument(fullName = VID_MAP_FILE_ARG_NAME,
+              shortName = VID_MAP_FILE_ARG_NAME,
+              doc = "Path to vid map JSON file to be created. Includes contig maps and INFO/FORMAT/FILTER" +
+                    "fields from headers. Default " + DEFAULT_VIDMAP_FILE_NAME +
+                    " file is written to GenomicsDB workspace",
+              optional = true)
+    private String vidMapJSONFileName = DEFAULT_VIDMAP_FILE_NAME;
+
+    @Argument(fullName = OVERWRITE_WORKSPACE_NAME,
+              shortName = OVERWRITE_WORKSPACE_NAME,
+              doc = "Will overwrite given workspace if it exists. " +
+                    "Otherwise a new workspace is created. " +
```

Comment:
This argument should only control whether to overwrite existing workspaces. The tool should always create a new workspace if it *doesn't* exist, regardless of the value of this argument.